src/online2bin/online2-tcp-nnet3-decode-faster_word_timestamps.cc

```diff
@@ -355,35 +357,38 @@ int main(int argc, char *argv[]) {
                 last_timestamp = last_timestamp + offset;
                 str_end = std::to_string(last_timestamp);
                 current_hypothesis = current_hypothesis + "{\"word\":" + "\"" + word + "\"," + "\"start\":" + str_start + "," + "\"end\":" + str_end + "},";
-              }
-              if (!current_hypothesis.empty()) {
-                current_hypothesis.erase(std::prev(current_hypothesis.end()));
+                word_count += 1;
+                global_block_end = str_end;
               }
 
+              if (word_count > 0) {
+              current_hypothesis.erase(std::prev(current_hypothesis.end()));
```

Comment:
Fix the formatting please (add indentation and align the brackets correctly). (And also everywhere else in the file where needed. I won't give more same comments on different places in this PR). The formatting should be the same in the whole code base:
```
if (true) {
    something;
}
else {
    something_else;
}
```